scripts/process_mentors.py

```diff
@@ -56,25 +56,19 @@ def process_row(row):
         for i, field in enumerate(CSV_FIELDS):
             if field:
                 mentor[field] = row[i]
-        im_name = mentor['firstNameAndLastName'].replace(' ', '').replace('(', '').replace(')', '')
+        mentor_id = mentor['firstNameAndLastName'].replace(' ', '').replace('(', '').replace(')', '')
```

Comment:
There should be some code to ensure that IDs are unique, unless we’re really confident.

Reply:
The script as it is currently makes the assumption that the image files, which are names after the person's name, are all unique `.jpg`, so this guarantees that the IDs are unique. I suppose we can work out how to deal with duplicate names when they arise... :P